docker/hadoop/common/hive-scripts/00_create_airlines.hql

```diff
@@ -1,8 +1,8 @@
 CREATE EXTERNAL TABLE IF NOT EXISTS AirlinesTest(
-  fYear STRING ,
-  fMonth STRING ,
-  fDayofMonth STRING ,
-  fDayOfWeek STRING ,
+  fYear VARCHAR(10) ,
```

Comment:
will need to rebuild images and bump version

Reply:
oh, thx!